shift-shift.go

```diff
@@ -213,13 +216,17 @@ func listenKeyboards(
 	display *C.Display,
 	keyFirst uint16, keySecond uint16,
 	printMode, quietMode bool, deviceMatch *regexp.Regexp,
-	scanOnce bool,
+	scanOnce bool, dblKeystroke bool, dblKeyTimeout int,
 ) {
 	var groupFirst, groupSecond bool
+	var t *time.Timer
 
 	inbox := make(chan Message, 8)
 	kbdLost := make(chan bool, 8)
 	kbdLost <- true // init
+	if dblKeystroke {
+		t = time.NewTimer(time.Duration(dblKeyTimeout * 1000 * 1000))
```

Comment:
There is a more canonical way to deal with time units in Go with using constants from the `time` package:
```go
time.Duration(dblKeyTimeout) * time.Millisecond)
```

Reply:
Okay, sure.